### DISCUSSION TITLE
More than a week still no reply from GitHub support on Account Suspension (Ticket ID: 2913646)

### DISCUSSION THREAD

**Discussion by Sahelisaha04** [community/community#135087] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**Subject:** Urgent: Account Suspension (Ticket ID: 2913646)

Dear GitHub Support Team,

I hope this message finds you well. My GitHub account username: **saheli-1**, was recently suspended after I submitted a discussion post about the 10th anniversary of the Student Developer Pack. This suspension has severely impacted my ongoing projects, access to essential tools, and my ability to contribute to open source.

I have already raised a support ticket (Ticket ID: **2913646**) more than a week ago but have not received a response yet. I’ve noticed that others with ticket numbers higher than mine (having same suspension issue) have already received replies, which has added to my concern.

Please assist me in reinstating my account as soon as possible. If there was any issue leading to the suspension, I am eager to address and resolve it promptly. Your urgent help in this matter would be greatly appreciated.

Thank you for your attention and support.

Best regards,  
saheli

**Comment by ebndev** [comment]
Hi @mavrickrishi,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by Sahelisaha04** [reply]
@ebndev It's been more than a week, and everyone keeps receiving the same response from GitHub staff, even though the ticket was opened months ago. My account was flagged without any notification, and I've been rejected from multiple jobs because they can't view my profile. This is incredibly frustrating. I raised a ticket more than a week ago, but nothing has been resolved. If I lose my current job over this, especially since all my company-related updates are in my repository, GitHub won't take responsibility. Is there any premium or expedited service that can help resolve this issue? I'm sorry, but I'm feeling helpless and out of options.

**Reply by Sahelisaha04** [reply]
@ebndev any reply
